### PR TITLE
Add License, CoC, and a footer

### PR DESCRIPTION
**Pull request by danfishgold**
A bunch of stuff we need to apply to [Netlify's open source plan](https://www.netlify.com/legal/open-source-policy):

- A license file (MIT)
- A code of conduct (which mainly refers to the one on our website)
- A footer with a link to Netlify (and a link to the repo)

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License Copyright (c) 2022 Dan Fishgold
```
Comment (katjam) [inline LICENSE]: Does it have to be a person or can it be Code Reading Club CIC ?

I mean I know it does not exist yet, but... I don't think that matters or if it does, then leave the CIC off - we are informally an established organisational body

Comment (katjam) [reply LICENSE]: I mean unless you want the (c) which is also fine.

**Review by katjam — APPROVED** [review]
Yup. Looks sound to me.

File: src/pages/SourceSelectionPage.tsx [hunk 2]
```diff
@@ -2,25 +2,29 @@ import React, { ChangeEvent, FormEvent, useCallback, useState } from 'react'
...
-        https://github.com/owner/repo/blob/908de054006934a071f770906119ce6d35a5a612/some/file/path.ext
-      </code>
-      <p>Paste it below:</p>
-      <Form />
+    <div className='home-page'>
+      <main>
+        <h1>Code Annotation Tool</h1>
+        <p>
+          Use this to annotate code in{' '}
+          <a href='https://code-reading.org'>Code Reading Clubs</a>.
```
Comment (katjam) [inline src/pages/SourceSelectionPage.tsx]:
```suggestion
          <a href='https://codereading.club'>Code Reading Clubs</a>.
```

Comment (danfishgold) [reply src/pages/SourceSelectionPage.tsx]: oh right thanks!

File: LICENSE (hunk 1 above)
Comment (danfishgold) [reply LICENSE]: the plan was to change it once we set up the CIC but I'll do that now :)

**Comment by katjam** [comment]
Merge away!

**Pull request merged**